profileservice/api/controllers/configs.py

```diff
@@ -22,17 +22,19 @@
 # this has to be have the file path to the folder for saving image files and other necessary files from rest service
 PROFILE_REST_STORAGE = os.getenv('REST_STORAGE', '/usr/src/app/rest')
 MONGO_PROFILE_URL = os.getenv('MONGO_PROFILE_URL', 'localhost:27017')
-MONGO_PII_URL = os.getenv('MONGO_PII_URL', 'localhost:27017')
+MONGO_PII_URL = os.getenv('MONGO_PROFILE_URL', 'localhost:27017')
```

Comment:
```suggestion
MONGO_PII_URL = os.getenv('MONGO_PII_URL', 'localhost:27017')
```
In Illinois instance, PII database is separate from Non-PII database marked in `MONGO_PROFILE_URL`. Also, from a Rokwire architecture perspective, we think it's a good practice to separate PII data from Non PII data.